src/RichText.js

```diff
@@ -34,6 +38,8 @@ const MARKDOWN_REGEX = {
 
 const USERNAME_REGEX = /@\S+:\S+/g;
 const ROOM_REGEX = /#\S+:\S+/g;
+let EMOJI_REGEX = null;
+window.EMOJI_REGEX = EMOJI_REGEX = new RegExp(emojione.unicodeRegexp, 'g');
```

Comment:
While putting it on window looks like debugging code, this regexp is different (should be) different from the one in EmojiProvider since this handles rendering of unicode emoji in the message composer (a feature which is disabled for now due to bugs in Draft.js)